docs/releasenotes/4.0.0.rst

```diff
@@ -102,6 +102,34 @@ can be rewritten to::
     IF    not ${session_created}    Create Session
     Skip If    not ${rows}
 
+New rule for misplaced not in conditions (#893)
+-----------------------------------------------
+
+Added new rule I0925 ``misplaced-not-in-condition`` that checks if the evaluated condition contains ``not`` operator
```

Comment:
What about `misplaced-not-condition`?

Reply:
I don't have strong preference though ``misplaced-not-condition`` sounds like whole 'not-type condition' was misplaced, and not just position of not in condition

Reply:
What about something like `suboptimal-negative-condition`? Can't find a better word than `suboptimal`, though... or maybe `misplaced-negative-condition`?

Reply:
``misplaced-negative-condition`` sounds better, I will go with it

Reply:
Sure, please also update release notes.